src/exception.cr

```diff
@@ -2,18 +2,39 @@ require "callstack"
 
 CallStack.skip(__FILE__)
 
+# Represents errors that occur during application execution.
+#
+# Descendants of class Exception are used to communicate between raise and
+# rescue statements in begin ... end blocks.
+# Exception objects carry information about the exception – its type (the
+# exception’s class name), an optional descriptive string, and
+# optional traceback information.
+# Exception subclasses may add additional information.
 class Exception
   getter message : String?
+  # Returns the previous exception at the time this exception was raised.
+  # This is useful for wrapping exceptions and retaining the original
+  # exception information.
   getter cause : Exception?
   property callstack : CallStack?
 
   def initialize(@message : String? = nil, @cause : Exception? = nil)
   end
 
+  # Returns any backtrace associated with the exception.
+  # The backtrace is an array of strings, each containing
+  # “0xAddress: Function at File Line Column”.
+  #
+  # See also: `CallStack#decode_backtrace`.
```

Comment:
`CallStack` is explicitly excluded from the docs so this "See also" probably won't help.